src/Controller/Controller.php

```diff
@@ -634,12 +642,12 @@ public function render(?string $template = null, ?string $layout = null): Respon
             return $this->response;
         }
 
-        if ($builder->getTemplate() === null && $this->request->getParam('action')) {
+        if ($builder->getTemplate() === null) {
```

Comment:
Can `getParam('action')` ever return null?

Reply:
I don't think that's possible in normal execution but even if it is, you would be just setting existing null value to again null, no harm done.